### PR TITLE
Add more uart description

### PR DESCRIPTION
**Pull request by andyp1per**

### CODE REVIEW

File: dev/source/docs/using-sitl-for-ardupilot-testing.rst [hunk 1]
```diff
@@ -431,9 +431,19 @@ To use a real serial device you can use a command like this:
 
 what that does it pass the --uartB argument to the ardupilot code,
 telling it to use /dev/ttyUSB0 instead of the normal internal simulated
-GPS for the 2nd UART.
+GPS for the 2nd UART. You can find the serial port mappings :ref:`here <common-pixhawk-serial-names>`
```
Comment (Hwurzburg) [inline dev/source/docs/using-sitl-for-ardupilot-testing.rst:434]: Probably should be more generalized for other bds...serial->uart is usually in the boards description in common_autopilots

Comment (andyp1per) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst:434]: I don't think this is right? The link I'm looking for is this one https://ardupilot.org/copter/docs/common-pixhawk-serial-names.html and the PX4_HAL column. They are synthetic values. I think the page is misnamed

Comment (tridge) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst:434]: this is not relevant for SITL

Comment (andyp1per) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst:434]: How about we add the uartA names etc to common-serial-options ?

**Comment by Hwurzburg** [comment]
Andy, from the discussions above, I think we can close this one....agree?

**Comment by andyp1per** [comment]
No, do not agree! How do I map from the non-obvious uart letters to serial numbers. That is what this is trying to address. Need a different table I guess.

**Comment by peterbarker** [comment]
Need a common-sitl-serial-names document?  I can't imagine it would be particularly complicated - there's not *that* many names we give serial ports in SITL.  As opposed to PixHawk/NuttX which had 6 IIRC!

**Comment by andyp1per** [comment]
@peterbarker exactly

**Comment by Hwurzburg** [comment]
if you would like for me to add it, I would need to know where to find the information to do so...the sim_vehicle help dialog does not seem to even mention the use of real serial devices, format for adding, or mapping of sim UARTx to sim SERIALx_ ports...

**Comment by andyp1per** [comment]
Ok I have cherry-picked @peterbarker 's SITL mappings and updated.

**Comment by Hwurzburg** [comment]
Made some changes, added to TOC, fixed some formats....merging..thanks....

**Pull request merged**